srcpkgs/nushell/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'nushell'
 pkgname=nushell
-version=0.33.0
+version=0.34.0
 revision=1
 build_style=cargo
 configure_args="--features=extra"
 hostmakedepends="pkg-config python3 libgit2-devel"
-makedepends="openssl-devel libgit2-devel libX11-devel libxcb-devel"
+makedepends="openssl-devel libgit2-devel libX11-devel libxcb-devel
+ libatomic-devel"
```

Comment:
Shouldn't this be added only for the `XBPS_TARGET_NO_ATOMIC8` case?

Reply:
The most recent push failed on i686 despite that arch having ATOMIC8, so I think it should be for all arch's.

Reply:
Did it fail because of network issues or with `couldn't find __atomic*` errors?

Reply:
The latter. I don't have a screenshot so I can't prove it, but it said it couldn't find `atomic*`.

Reply:
Just mention it in the commit message, then :)

`add libatomic for all targets, since build system always uses it`